beacon-chain/forkchoice/protoarray/types.go

```diff
@@ -12,6 +12,8 @@ type ForkChoice struct {
 	votes     []Vote // tracks individual validator's last vote.
 	votesLock sync.RWMutex
 	balances  []uint64 // tracks individual validator's last justified balances.
+	//nolint:deadcode
```

Comment:
```suggestion
```

Why is this lint suppression here?

Reply:
Oh nice catch, it's because of the first PR that didn't use this.